django_web_exceptions-0.1.4.dist-info/DESCRIPTION.rst,sha256=tvNaFdYf4CcxmWNkGZ0GKiCj4RVUVvIjVRWtbZprQgI,3386
django_web_exceptions-0.1.4.dist-info/METADATA,sha256=lcY9vv90KUmdEeXC3xJtGPBxsB0_wHjI8fzrl8DRGwA,4135
django_web_exceptions-0.1.4.dist-info/RECORD,,
django_web_exceptions-0.1.4.dist-info/WHEEL,sha256=o2k-Qa-RMNIJmUdIc7KU6VWR_ErNRbWNlxDIpl7lm34,110
django_web_exceptions-0.1.4.dist-info/metadata.json,sha256=TiWIAbkPabm7Os-KazktfBoyPVSl_IEpp_yqtEHqCs8,929
django_web_exceptions-0.1.4.dist-info/top_level.txt,sha256=wLetahU8QNvabnmEkZSF_ILDGUwLc6SIQW0FweaCJdA,15
web_exceptions/__init__.py,sha256=aBEbDvx4LMg8A1TJJR6dEHu8rQODVin528hLS_EDvuA,22
web_exceptions/exceptions.py,sha256=CIS_4-qP-aCZTg1QSRv8VwtMkUq0T8Uz6xH8f3ELNNQ,7082
web_exceptions/middleware.py,sha256=RRC1OVi2iLlSzLaICtxe22gRUjxN6BqBG2-I7uWl0Ao,709
